README.md

```diff
@@ -13,58 +13,15 @@
 __DC/OS SDK__ is a collection of tools, libraries, and documentation for easy integration of technologies such as Kafka, Cassandra, HDFS, Spark, and TensorFlow with DC/OS.
 
 ---
-### Understanding the Hello World Service Specification
+### Quick Start
 
-The service specification declaratively defines the `helloworld` service:
+To create a new service based on the SDK:
+1. read the [developer guide](https://mesosphere.github.io/dcos-commons/developer-guide/) and the [FAQ](https://mesosphere.github.io/dcos-commons/faq/)
+1. either install and use the [dcosdev tool](https://github.com/mesosphere/dcosdev) or follow the example of [hello-world framework](frameworks/helloworld)
```

Comment:
I think you meant a numbered list here.

```suggestion
2. either install and use the [dcosdev tool](https://github.com/mesosphere/dcosdev) or follow the example of [hello-world framework](frameworks/helloworld)
```

Reply:
I did mean a numbered list, but using all ones is a common pattern in markdown, lets you take the renderer take care about keeping the numbers right.
See [the MD guide](https://www.markdownguide.org/basic-syntax/#lists) and that [it actually renderes correctly](https://github.com/mesosphere/dcos-commons/tree/DCOS-52239/fix#quick-start)